arches/install/requirements.txt

```diff
@@ -24,7 +24,7 @@ requests[security]>=2.31.0
 python-slugify==7.0.0
 pillow>=7.0.0
 arcgis2geojson==2.0.0
-openpyxl==3.0.10
+openpyxl==3.1.2
```

Comment:
Would you mind also keeping the summary of upgraded versions in arches/releases/7.6.0.md up-to-date with this change?

Reply:
Yes, of course.